app/reconciliation.go

```diff
@@ -124,6 +126,9 @@ func (app *App) reconPaymentChanges(ctx sdk.Context, paymentIavl *iavl.Store) bo
 					ctx.Logger().Error("fail to unmarshal stream record", "err", err.Error())
 				} else {
 					flowCurrent = flowCurrent.Add(sr.NetflowRate)
+
+					j, _ := json.Marshal(sr)
```

Comment:
do we need to handle this error

Reply:
no need, just for debugging when there are issues.
will remove when formal release